### PR TITLE
18630 seed real roles

### PR DESCRIPTION
**Pull request by nnorbert**
seed real roles

### CODE REVIEW

File: packages/cli/src/commands/database/ogcio/common-rbac.ts [hunk 1]
```diff
@@ -145,17 +148,30 @@ export const fillScopesGroup = <
...
     }
 
-    addScopeToLists(fullLists, resource, action, resourceId, subject);
+    addScopeToLists(fullLists, resource, action, resourceId, prefix);
   }
 
   for (const resource of seeder.entities ?? []) {
     for (const action of seeder.actions ?? []) {
-      addScopeToLists(fullLists, resource, action, resourceId);
+      addScopeToLists(fullLists, resource, action, resourceId, seeder.prefix);
```
Comment (peschina) [inline packages/cli/src/commands/database/ogcio/common-rbac.ts:174]: I'm confused about `seeder.prefix`, will it ever be defined?

**Review by peschina — APPROVED** [review]
Looks good overall

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-dev.json [hunk 2]
```diff
@@ -7,6 +7,28 @@
                 "id": "ogcio"
             }
         ],
+        "organization_permissions": [
+            {
+                "specific_permissions": [
+                    "payments:provider:*",
+                    "payments:payment_request:*",
+                    "payments:payment_request.public:read",
+                    "payments:transaction:*"
```
[the remark this answers is not in the document]
Comment (nnorbert) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-dev.json]: That is programmatically not possible to do by a public servant, so no specific permission is needed. @alfonsograziano please confirm.

File: packages/cli/src/commands/database/ogcio/common-rbac.ts (hunk 1 above)
Comment (nnorbert) [reply packages/cli/src/commands/database/ogcio/common-rbac.ts]: Prefix is not relevant anymore, we changed the whole code.

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-dev.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (alfonsograziano) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-dev.json]: It is not possible to create a transaction so far for public servants, so the permission should be fine. We might reconsider that in the future, but for now we should be safe like this :)

**Review by alfonsograziano — APPROVED** [review]
LGTM

**Pull request merged**